cmd/kraft/run/runner_project.go

```diff
@@ -38,14 +38,21 @@ func (runner *runnerProject) String() string {
 func (runner *runnerProject) Runnable(ctx context.Context, opts *Run, args ...string) (bool, error) {
 	var err error
 
+	cwd, err := os.Getwd()
+	if err != nil {
+		return false, err
```

Comment:
Let's return the error with its context for easier troubleshooting:

```suggestion
		return false, fmt.Errorf("getting current working directory: %w", err)
```